xdem/__init__.py

```diff
@@ -1,5 +1 @@
-from . import coreg
-from . import spatial_tools
-from . import dem
-from . import examples
-from . import spstats
\ No newline at end of file
+from . import coreg, dem, examples, spatial_tools, volume, spstats
```

Comment:
good point here haha

Reply:
I just let my formatters go nuts! Apparently this is "pep8"-y